oscarapi/utils.py

```diff
@@ -18,10 +18,11 @@ def overridable(name, default):
 
 
 class OscarSerializer(object):
-    field_mapping = dict(
-        serializers.ModelSerializer.serializer_field_mapping, **{
-            oscar.models.fields.NullCharField: serializers.CharField
-        })
+    field_mapping = serializers.ModelSerializer.serializer_field_mapping.copy()
+    null_char_field_mapping = {
```

Comment:
Please don't assign to a variable first, just write:
```
field_mapping.update({
    oscar.models.fields.NullCharField: serializers.CharField
})
```

That way we can just update the field mapping if that is required for more fields.
If we would have to, the name of the intermediate variable wouldn't make any sense either.